s3prl/downstream/a2o-vc-vcc2020/config.yaml

```diff
@@ -0,0 +1,58 @@
+runner:
+  total_steps: 10000
+  gradient_clipping: 1
+  gradient_accumulate_steps: 1
+  max_keep: 10
+  eval_dataloaders:
+    - dev
+    - test
+  log_step: 100
+  eval_step: 1000
+  save_step: 1000
+
+  # debug
+  # log_step: 10
+  # eval_step: 50
+  # save_step: 50
+
+optimizer:
+  name: AdamW
+  lr: 1.0e-4
+
+# comment the whole scheduler config block
+# to disable learning rate scheduling
+scheduler:
+  name: linear_schedule_with_warmup
+  num_warmup_steps: 4000
+
+downstream_expert:
```

Comment:
I think it might be nice to put target speaker configuration inside this `downstream_expert` field and give an dummy example like `trgspk: TEF1  # downstream/a2o-vc-vcc2020/data/lists"
`

Reply:
Sure. I'll change this.